XamarinApp/XamarinApp/ViewModels/ApiDataPageViewModel.cs

```diff
@@ -0,0 +1,81 @@
+using Prism.AppModel;
+using Prism.Commands;
+using Prism.Mvvm;
+using Prism.Navigation;
+using Prism.Services;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Xamarin.CommunityToolkit.ObjectModel;
+using Xamarin.Forms;
+using XamarinApp.Resx;
+using XamarinApp.Services.Interfaces;
+using static XamarinApp.Models.ApiModel;
+
+namespace XamarinApp.ViewModels
+{
+  public class ApiDataPageViewModel : ViewModelBase, INavigatedAware, IPageLifecycleAware
```

Comment:
you can add these two interface in ViewmodelBase class, and override it wherever you needed.

Reply:
These are like default interface it can be added only on required viewModel when we added on ViewModelBase we should implement that interfaces in all the palaces(ViewModel)

Reply:
Added